packages/config-transformer/build/config/config-downgrade.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Rector\Set\ValueObject\DowngradeSetList;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(DowngradeSetList::PHP_80);
+    $containerConfigurator->import(DowngradeSetList::PHP_74);
+    $containerConfigurator->import(DowngradeSetList::PHP_73);
+    $containerConfigurator->import(DowngradeSetList::PHP_72);
+    $containerConfigurator->import(DowngradeSetList::PHP_71);
```

Comment:
You want to downgrade to PHP 7.1, right? If so, notice that this line will downgrade to 7.0

Reply:
Good to konw, thanks :+1: